solcx/install.py

```diff
@@ -609,10 +609,15 @@ def _install_solc_windows(
 
     temp_path = _get_temp_folder()
     content = _download_solc(download, show_progress)
-    with zipfile.ZipFile(BytesIO(content)) as zf:
-        zf.extractall(str(temp_path))
 
-    temp_path.rename(install_path)
+    if filename.endswith("exe"):
+        with open(install_path, "wb") as fp:
+            fp.write(content)
```

Comment:
I think the issue here is that on windows systems, solc is placed in a unique folder for each installation.  This is required because zipped versions include a separate DLL file.

The simplest approach would then be to use the same file structure for these new non-zipped executables:

```suggestion
    if filename.suffix == ".exe":
        install_path.mkdir()
        with open(install_path.joinpath("solc.exe"), "wb") as fp:
            fp.write(content)
```

Reply:
sure, do you have a way to test it? I don't have a windows box :(

Reply:
`AttributeError: 'str' object has no attribute 'suffix'`

Reply:
ooooh...  my bad! `Path(filename).suffix`